ui/core.js

```diff
@@ -157,13 +157,13 @@ $.extend( $.expr[ ":" ], {
 		},
 
 	focusable: function( element ) {
-		return focusable( element, !isNaN( $.attr( element, "tabindex" ) ) );
+		return focusable( element, $.attr( element, "tabindex" ) != null );
```

Comment:
So the presence of a "tabindex" attribute is all that matters, independent of its value? In that case, :+1:.